pensjonsmaler/src/main/kotlin/no/nav/pensjon/brev/maler/vedlegg/pdf/InformasjonOmP1PDFDto.kt

```diff
@@ -0,0 +1,19 @@
+package no.nav.pensjon.brev.maler.vedlegg.pdf
+
+import no.nav.pensjon.brev.template.LangBokmalEnglish
+import no.nav.pensjon.brev.template.Language
+import no.nav.pensjon.brev.template.dsl.newText
+import no.nav.pensjon.brev.template.vedlegg.createAttachmentPDF
+import no.nav.pensjon.brevbaker.api.model.EmptyPDFVedleggData
+
+val informasjonOmP1Vedlegg = createAttachmentPDF<LangBokmalEnglish, EmptyPDFVedleggData>(
+    title = listOf(
+        newText(
```

Comment:
Ser at vi burde endre på newText til å bruke det nye LiteralOrExpressionBuilder-konseptet, men det er en jobb for en annen dag.